pkg/v1/tkg/web/src/app/shared/contextual-help/build-script/convert.js

```diff
@@ -0,0 +1,104 @@
+const fs = require('fs');
+const path = require('path');
+const axios = require('axios');
+const showdown = require('showdown');
+const pathData = require('./docPath.json');
+
+
+const CWD = process.cwd();
+
+const saveFile = (filePathDest, content) => {
+    fs.writeFile(filePathDest, content, err => {
+        if (err) {
+            console.error(err);
+        }
+    });
+};
+const convertFile = (data) => {
+    const converter = new showdown.Converter();
+    return converter.makeHtml(data); 
+}
+const convertLocalFile = (filePathSrc, filePathDest) => {
+    fs.readFile(CWD + filePathSrc, 'utf8', (err, data) => {
+        if (err) {
+            console.error(err);
+            return;
+        }
+        saveFile(CWD + filePathDest, convertFile(data));
+    });
+}
+
+const convertLocalFiles = (pathInfo) => {
+    const srcDir = pathInfo.source;
+    const destDir = pathInfo.destination;
+
+    fs.readdir(CWD + srcDir, (err, files) => {
+        if (err) {
+            console.error(err);
+            return;
+        }
+        files.forEach(file => {
+            convertLocalFile(
+                srcDir + file,
+                destDir + file.replace('.md', '.html')
+            );
+        });
+    });
+}
+const getDestination = (pathInfo, fromLocal) => {
+    const src = pathInfo.source;
+    const dest = pathInfo.destination;
+
+    let statsSrc = null;
+    if (fromLocal) {
+        statsSrc = fs.statSync(CWD + src);
+
+    }
+    
+    if (dest.substr(-5) === '.html') { // convert one file with a different name.
+        if (statsSrc && !statsSrc.isFile()) {
+            console.error('A directory can not be converted to a file');
+        }
+        return pathInfo.destination;
+    }
+
+    if (!fromLocal || statsSrc.isFile()) { //convert one file to a directory
+        return dest + path.basename(src).replace('.md', '.html');
+    }
+    return ''; // save all files in src directory to destination directory. 
+}
+const isFromLocal = (arg) => {
```

Comment:
nit: I'd suggest isLocalPath() instead of isFromLocal().

Reply:
renamed